dev/io.openliberty.microprofile.lra.1.0.internal/src/io/openliberty/microprofile/lra/internal/LraFilterComponent.java

```diff
@@ -63,8 +63,9 @@ protected void activate(Map<String, Object> properties) throws LraException {
             URI coord = new URI(coordString);
             NarayanaLRAClient.setDefaultCoordinatorEndpoint(coord);
         } catch (URISyntaxException e) {
-            // TODO Not sure how to handle config problems yet
-            Tr.error(tc, "Things went wrong");
+            if (TraceComponent.isAnyTracingEnabled() && tc.isErrorEnabled()) {
+                Tr.error(tc, "Invalid LRA coordinator URI: '" + coordString + "'");
+            }
```

Comment:
Don't check `isAnyTracingEnabled()` when logging errors.

Checking `isErrorEnabled()` is not wrong but usually unnecessary, you can assume that errors and warnings are always enabled. The check here would only be for performance anyway.